docs/site/tutorials/authentication/Authentication-Tutorial.md

```diff
@@ -0,0 +1,974 @@
+---
+lang: en
+title: 'How to secure your LoopBack 4 application with JWT authentication'
+keywords: LoopBack 4.0, LoopBack 4, Authentication, Tutorial
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/authentication-tutorial.html
```

Comment:
i think it has to match the file name case so either both should be `authentication-tutorial` or `Authentication-Tutorial`

Reply:
It works because I have tested the generated HTML and loaded it into a local docs server, but I will make sure they match.